tasking/pulp/tasking/celery_app.py

```diff
@@ -92,6 +94,7 @@ def get_resource_manager_lock(name):
     :param name:   The hostname of the worker
     :type  name:   basestring
     """
+    from pulp.app.models.task import TaskLock, Worker
```

Comment:
I agree, this seems more fragile than just the 1 line import. I'm ok if there are no changes in this area.